source/documentation/datatypes/datatypes.md

```diff
@@ -0,0 +1,85 @@
+## Boolean
+
+A datatype which can have one of two states, `True` and `False`. 
```

Comment:
I think datatypes in this documentation have to explain how are they returned in the API. And that is as strings so, I would suggest you do an into explaining that and then for each datatype explain what it is and what does it look like in its string form.

E.g. A boolean in its string form is either "true" or "false".